policy/modules/system/systemd.te

```diff
@@ -300,13 +312,18 @@ fs_register_binary_executable_type(systemd_binfmt_t)
 # generic generator local policy
 #
 
-corecmd_search_bin(systemd_generator_type)
+allow systemd_generator_type self:process setfscreate;
+
+corecmd_exec_bin(systemd_generator_type)
+corecmd_exec_shell(systemd_generator_type)
```

Comment:
I don't think this should be applied to all generators.

Reply:
After the discussion on the mailing list, should I revert this to its original form ?